configs/next/packages/gcc/stage_3

```diff
@@ -35,6 +35,23 @@ atcfg_pre_hacks() {
 		[[ -h ${Z%\.a}_s.a ]] && rm -f ${Z%\.a}_s.a
 		[[ -h ${Z%\.a}_eh.a ]] && rm -f ${Z%\.a}_eh.a
 	done
+
```

Comment:
Is the code above still needed?  "Clean static lib hack done on gcc stage 2", now that stage 2 is defunct?

Reply:
No it isn't needed anymore.  I missed this.